src/PrestaShopBundle/Resources/config/services/bundle/routing.yml

```diff
@@ -4,3 +4,8 @@ services:
         arguments:
           - '@=service("prestashop.module_kernel.repository").getActiveModulesPaths()'
         tags: [routing.loader]
+
+    prestashop.bundle.routing.legacy_url_converter:
+        class: 'PrestaShopBundle\Routing\LegacyUrlConverter'
+        arguments:
+            $router: '@router'
```

Comment:
😮 I did not know this syntax for SF services
👍

Reply:
FYI: https://symfony.com/blog/new-in-symfony-3-4-local-service-binding